docs/book/getting-started/deploying-zenml/helm.md

```diff
@@ -390,6 +441,210 @@ you can map the hostname `zenml.<subdomain>` to the Ingress controller's IP
 address or hostname. Then, simply use the new hostname to expose ZenML at the
 root URL path.
 
+### Using the AWS Secrets Manager as a secrets store backend
+
+Unless explicitly disabled or configured otherwise, the ZenML server will use
+the SQL database as a secrets store backend. If you want to use the AWS Secrets
+Manager instead, you need to configure it in the Helm values. Depending on where
+you deploy your ZenML server and how your Kubernetes cluster is configured, you
+may also need to provide AWS credentials needed to access the AWS Secrets
+Manager API:
+
+```yaml
+ zenml:
+
+  # ...
+
+  # Secrets store settings. This is used to store centralized secrets.
+  secretsStore:
+
+    # Set to false to disable the secrets store.
+    enabled: true
+
+    # The type of the secrets store
+    type: aws
+
+    # Configuration for the AWS Secrets Manager secrets store
+    aws:
+
+      # The AWS region to use. This must be set to the region where the AWS
+      # Secrets Manager service that you want to use is located.
+      region_name: us-east-1
+
+      # The AWS credentials to use to authenticate with the AWS Secrets
+      # Manager instance. You can omit these if you are running the ZenML server
+      # in an AWS EKS cluster that has an IAM role attached to it that has
+      # permissions to access the AWS Secrets Manager instance.
+      aws_access_key_id: <your AWS access key ID>
+      aws_secret_access_key: <your AWS secret access key>
+      aws_session_token: <your AWS session token>
+```
+
+### Using the GCP Secrets Manager as a secrets store backend
+
+Unless explicitly disabled or configured otherwise, the ZenML server will use
+the SQL database as a secrets store backend. If you want to use the GCP Secrets
+Manager instead, you need to configure it in the Helm values. Depending on where
+you deploy your ZenML server and how your Kubernetes cluster is configured, you
+may also need to provide GCP credentials needed to access the GCP Secrets
+Manager API:
+
+```yaml
+ zenml:
+
+  # ...
+
+  # Secrets store settings. This is used to store centralized secrets.
+  secretsStore:
+
+    # Set to false to disable the secrets store.
+    enabled: true
+
+    # The type of the secrets store
+    type: gcp
+
+    # Configuration for the GCP Secrets Manager secrets store
+    gcp:
+
+      # The GCP project ID to use. This must be set to the project ID where the
+      # GCP Secrets Manager service that you want to use is located.
+      project_id: us-east-1
```

Comment:
```suggestion
      project_id: my-zenml-project
```

Confusing to use a region-name value as a project id, even if technically valid...

Reply:
You're right, it was a copy-paste mishap.